docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,145 @@
 ZWO Camera
 ==========
 
+This service operates a ZWO camera. There are currently four different types of ZWO cameras being used on the testbed:
+- ZWO ASI533MM (Science camera, https://www.zwoastro.com/product/asi533mm-mc/)
+- ZWO ASI290MM (TA camera, https://agenaastro.com/zwo-asi290mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI178MM (MOWFS Camera and Pupil Camera, https://agenaastro.com/zwo-asi178mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI1600MM (Phase Retrieval Camera, https://agenaastro.com/zwo-asi1600mm-p-cmos-monochrome-astronomy-imaging-camera-pro.html)
```

Comment:
The documentation should not refer to a specific testbed. Instead, there should just be a list of concrete devices (the ones you list here) that have been tested and used with catkit2 so far.